package.json

```diff
@@ -38,9 +38,9 @@
     "hash": "./scripts/hash.sh",
     "package": "./scripts/package.sh",
     "watch": "rm -rf dist && run-p watch:*",
-    "watch-with-vscode": "run-p watch watch-vscode",
+    "watch-with-vscode": "export DEV_VSCODE=true && run-p watch watch-vscode",
```

Comment:
Is this the same as `DEV_VSCODE=true run-p watch watch-vscode`?

Reply:
They are same here, but I think yours is better.
When users who want to copy this command to shell and execute it, the `export` will make a effect in the future.
I have modified it, thank you so much!

Reply:
Thanks